assets/js/components/Work/Tabs/Administrative.test.jsx

```diff
@@ -69,6 +69,8 @@ describe("Work Administrative tab component", () => {
       expect(getByText(/Another Project Description/i)).toBeInTheDocument();
       expect(getByText(/Project Cycle Name/i)).toBeInTheDocument();
       expect(getByText(/Started/i)).toBeInTheDocument();
+      expect(getByText(/Collection 1232432 Name/i)).toBeInTheDocument();
```

Comment:
Just to refine a pattern, we don't need the `.toBeInTheDocument()` chained function anymore.  `expect` will handle the truthiness of the return value.  Mind wiping those off this test?   I'm doing it as I come across tests which still use it.